src/messages/LabeledMsg.ts

```diff
@@ -35,6 +42,8 @@ export interface LabeledMsg<T, V> extends PartialMsg<T, V> {
   protocol: 'iframe-coordinator';
   /** library version */
   version: string;
+  /** So that nested iframe-coordinators can ignore messages that don't apply */
+  direction?: MessageDirection;
```

Comment:
Are we sure this actually needs to be optional? From the perspective of typechecking in this library it should always be set.

Reply:
With a host and client, and only one of them has updated iframe-coordinator, I believe it would validate incoming messages from the non-upgraded version and say they are invalid if this was not optional

Reply:
Oh you mean the type, not the validation, which happens in directionDecoder. I'll try that.

Reply:
Ah, the decoder needing it makes sense. I was thinking that it wouldn't matter with old clients since at runtime all the type information is gone anyway, but the decoder definitely needs to handle the undefined case. Thanks for clarifying!